### PR TITLE
Move template fields poll_voted_ids, option_ids, vote_ids into meeting_user.

### PR DESCRIPTION
**Pull request by reiterl**
Update model, actions and tests.

### CODE REVIEW

**Comment by reiterl** [comment]
I skipped some poll.vote tests, because they need a new vote-service.

File: openslides_backend/action/actions/poll/mixins.py [hunk 1]
```diff
@@ -85,8 +85,8 @@ def on_stop(self, instance: Dict[str, Any]) -> None:
             vote_weight = Decimal(ballot["weight"])
             votesvalid += vote_weight
             vote_template = {"user_token": user_token}
-            if "vote_user_id" in ballot:
-                vote_template["user_id"] = ballot["vote_user_id"]
+            if "vote_meeting_user_id" in ballot:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/poll/mixins.py:88]: Changed.

File: global/meta/models.yml [hunk 2]
```diff
@@ -321,27 +321,6 @@ user:
       type: relation-list
       to: group/user_ids
     restriction_mode: A
-  poll_voted_$_ids:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:324]: Moved the three fields into the user collection again.

File: openslides_backend/action/actions/vote/create.py [hunk 3]
```diff
@@ -43,15 +43,12 @@ def prefetch(self, action_data: ActionData) -> None:
             use_changed_models=False,
         )
         fields = [
-            "vote_$_ids",
```
Comment (r-peschke) [inline openslides_backend/action/actions/vote/create.py:46]: Shouldn't we use instead the new fields? But I don't know, what they are for!

**Review by r-peschke — COMMENTED** [review]
What are the fields for in vote.create?
Do we really have to skip all the vote-tests?

File: openslides_backend/action/actions/vote/create.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/action/actions/vote/create.py:46]: Add the new fields here.

File: openslides_backend/action/actions/vote/create.py [hunk 4]
```diff
@@ -43,15 +43,13 @@ def prefetch(self, action_data: ActionData) -> None:
...
-            "poll_voted_$_ids",
-            "vote_delegated_vote_$_ids",
+            "vote_ids",
+            "poll_voted_ids" "vote_delegated_vote_$_ids",
```
Comment (r-peschke) [inline openslides_backend/action/actions/vote/create.py:47]: I miss something;-)

File: tests/system/action/poll/test_stop.py [hunk 5]
```diff
@@ -253,6 +253,7 @@ def test_stop_permissions(self) -> None:
             Permissions.Poll.CAN_MANAGE,
         )
 
+    @pytest.mark.skip
```
Comment (r-peschke) [inline tests/system/action/poll/test_stop.py:256]: Should be only a different number of database calls

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/vote/create.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/action/actions/vote/create.py]: Add the comma.

File: tests/system/action/poll/test_stop.py (hunk 5 above)
Comment (reiterl) [reply tests/system/action/poll/test_stop.py:256]: The test doesn't work right. Skip it for now.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**